src/languages/swift.js

```diff
@@ -351,7 +351,7 @@ export default function(hljs) {
       lookahead(concat(Swift.identifier, /\s*:/)),
       lookahead(concat(Swift.identifier, /\s+/, Swift.identifier, /\s*:/))
     ),
-    end: lookahead(/:/),
+    end: /:/,
```

Comment:
No good reason not to consume this that I can see, so we just do that, since it's simpler.

Reply:
Yes, that should be fine. I didn't consume it just it case I need it to detect a type annotation (which always starts with `:`), but I managed to get proper highlighting without having to do that.

Reply:
>  I didn't consume it just it case I need it to detect a type annotation

We really need look-behind. :(  Lots of complexity dealing with these kind of things without it.